cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -94,12 +94,11 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
     generateConsole = false
 
     prepareAndTestQuery(
-      title = "List constraints",
+      title = "Example of listing constraints",
       text =
-        """include::list-constraints-table-columns.asciidoc[]
-          |
-          |Note that the old deprecated built-in procedures for listing constraints, such as `db.constraints`, work as before and are not affected by the
-          |<<administration-security-administration-database-constraints, `SHOW CONSTRAINTS` privilege>>.""".stripMargin,
+        """
+          |To list all constraints with the brief output columns, the `SHOW CONSTRAINTS` command can be used.
```

Comment:
I'd like to do some smaller changes to this example to make it closer to how the index example works (mentioning the filtering and `VERBOSE`). I can update and push the changes since it differs slightly from index and can't be copied over)

Reply:
Sounds good, please do!